diff --git a/node_modules/class-validator/validation/ValidationExecutor.js b/node_modules/class-validator/validation/ValidationExecutor.js
index f4f876e..bbee75b 100644
--- a/node_modules/class-validator/validation/ValidationExecutor.js
+++ b/node_modules/class-validator/validation/ValidationExecutor.js
@@ -31,15 +31,6 @@ var ValidationExecutor = /** @class */ (function () {
     // -------------------------------------------------------------------------
     ValidationExecutor.prototype.execute = function (object, targetSchema, validationErrors) {
         var _this = this;
-        /**
-         * If there is no metadata registered it means possibly the dependencies are not flatterned and
-         * more than one instance is used.
-         *
-         * TODO: This needs proper handling, forcing to use the same container or some other proper solution.
-         */
-        if (!this.metadataStorage.hasValidationMetaData) {
-            console.warn("No metadata found. There is more than once class-validator version installed probably. You need to flatten your dependencies.");
-        }
         var groups = this.validatorOptions ? this.validatorOptions.groups : undefined;
         var targetMetadatas = this.metadataStorage.getTargetValidationMetadatas(object.constructor, targetSchema, groups);
         var groupedMetadatas = this.metadataStorage.groupByPropertyName(targetMetadatas);
